internal/cmd/server.go

```diff
@@ -104,7 +104,7 @@ The kernel is used to run long running processes like shells and interacting wit
 			parserv1.RegisterParserServiceServer(server, editorservice.NewParserServiceServer(logger))
 			projectv1.RegisterProjectServiceServer(server, projectservice.NewProjectServiceServer(logger))
 			if enableRunner {
-				runnerServicev1, err := runner.NewRunnerService(logger)
+				runnerServicev1, err := runner.NewRunnerService(logger, addr, tlsDir)
```

Comment:
don't quite understand why we need this

Reply:
They were added to be available later as environment variables when a new session is created.

Reply:
https://github.com/stateful/runme/pull/532/files#diff-50b92da422aa4c0c738d7a625064364d14c03cafd8b1048eefd6fb9ae10227d9R96

Reply:
NVM, Now I understand that variables are being injected in the extension.

![image](https://github.com/stateful/runme/assets/43882/6d29f271-200c-4324-bbf0-3479b5913ce2)

Reply:
Removing, you're right.